src/index.html

```diff
@@ -1,15 +1,120 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="POST"
+        onsubmit="onSubmit()">
+      <fieldset class="fiedset-block">
+        <legend>Personal information</legend>
+        <div class="fiedset-block__input">
+          <label for="surname">Surname:</label>
+          <input type="text" name="surname"
+            id="surname" autocomplete="off"
+            required>
```

Comment:
переноси каждый атрибут на новую строку
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
            required
          >
```